kubejs/server_scripts/tfg/tags.js

```diff
@@ -947,6 +953,23 @@ const registerTFGBlockTags = (event) => {
 	event.add('buildinggadgets2:deny', 'tfg:geyser_source_small')
 	event.add('ae2:blacklisted/spatial', 'tfg:geyser_source_small')
 
+	//#region Fision Components
+    var COMPONENTS = 'deafission:components';
```

Comment:
this feels like it should be in its own deafission folder, but we can do that in another review